service/iot-service-samples/pnp-service-sample/pom.xml

```diff
@@ -10,6 +10,14 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>pnp-service-sample</artifactId>
+    <dependencies>
```

Comment:
I see these samples as 2 separate java projects, each having their own pom file. That way, they are completely independent of each other.

Reply:
I'm not how that helps. In C# you need a csproj to run it right? But here it was easy enough to just run any file. What will be gain by separating them? I feel we will have a lot of fragmented code and these belong together.